tests/boot.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2014 SUSE Linux GmbH
+# Copyright (C) 2014-2017 SUSE LLC
```

Comment:
You mean 2018 right?

Reply:
No, because I did not change the files in 2018, the commits were done more than a year ago